pkgs/development/libraries/qca/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchurl, cmake, openssl, pkg-config, qtbase, qt5compat ? null }:
+
+let
+  isQt6 = lib.versions.major qtbase.version == "6";
```

Comment:
We should move that into some common shared lib thing between qt things.

Reply:
Unfortunately it's not very consistent between libraries.